store/sqlstore/upgrade.go

```diff
@@ -504,3 +506,11 @@ func UpgradeDatabaseToVersion54(sqlStore SqlStore) {
 		saveSchemaVersion(sqlStore, VERSION_5_4_0)
 	}
 }
+
+func UpgradeDatabaseToVersion55(sqlStore SqlStore) {
+	// TODO: Uncomment following condition when version 5.5.0 is released
+	// if shouldPerformUpgrade(sqlStore, VERSION_5_4_0, VERSION_5_5_0) {
+
+	// 	saveSchemaVersion(sqlStore, VERSION_5_3_0)
```

Comment:
Even though it's still a comment this should be `saveSchemaVersion(sqlStore, VERSION_5_5_0)` right?

Reply:
thanks! fixed